blockchain-source/src/cardano/time.rs

```diff
@@ -0,0 +1,71 @@
+const EPOCH_LENGTH_IN_SECONDS: u64 = 43200;
+const BYRON_SLOT_DURATION: u64 = 20;
+
+#[derive(Debug, Clone, serde::Deserialize)]
+pub struct Era {
```

Comment:
are we using the Era anywhere? in dcspark-core I see we only set it as a config parameter, but never read it

Reply:
I'm using the `absolute_slot_to_epoch` in carp, and the `compute_timestamp` is used in the bridge. I was a bit hesitant to port the entire module over, but I feel like it's better to keep it together.

That part is a bit weird though. There is a comment somewhere else that relates to this.